### DISCUSSION TITLE
The web takes a long time to load every time it goes in, and I hope to release an Android app that can be cached[Kavita] Idea / Feature Submission

### DISCUSSION THREAD

**Discussion by hausen1012** [Kareadita/Kavita#2755] [Ideas]
### Idea Category

API

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
I'm going to close this. This feature request doesn't make much sense nor will anyone be able to understand and upvote it. Feel free to reach out to support if you're having issues.
